qiskit/algorithms/quantum_time_evolution/variational/var_qrte.py

```diff
@@ -108,9 +112,8 @@ def evolve(
             hamiltonian_value_dict, list(initial_state.parameters)
         )
 
-        operator_coefficient = 1.0j
+        # operator_coefficient = 1.0j # TODO Remove
```

Comment:
Please remove.

Reply:
Please make sure that you remove this in the VarQITE code as well.
Did it now for VarQRTE.